tst/EnergyPlus/unit/HeatBalanceManager.unit.cc

```diff
@@ -1278,6 +1278,24 @@ TEST_F(EnergyPlusFixture, HeatBalanceManager_TestZonePropertyLocalEnv)
     EXPECT_EQ(20.0, Zone(1).OutWetBulbTemp);
     EXPECT_EQ(1.5, Zone(1).WindSpeed);
     EXPECT_EQ(90.0, Zone(1).WindDir);
+
+    // Add a test for #7308 without inputs of schedule names
+    DataLoopNode::Node(1).OutAirDryBulbSchedNum = 0;
+    DataLoopNode::Node(1).OutAirWetBulbSchedNum = 0;
+    DataLoopNode::Node(1).OutAirWindSpeedSchedNum = 0;
+    DataLoopNode::Node(1).OutAirWindDirSchedNum = 0;
+    DataEnvironment::OutDryBulbTemp = 25.0;
+    DataEnvironment::OutWetBulbTemp = 20.0;
+    DataEnvironment::WindSpeed = 1.5;
+    DataEnvironment::WindDir = 90.0;
```

Comment:
Poor unit test, the outdoor values (25, 20, 1.5 and 90).

```
    ScheduleManager::Schedule(1).CurrentValue = 25.0;
    ScheduleManager::Schedule(2).CurrentValue = 20.0;
    ScheduleManager::Schedule(3).CurrentValue = 1.5;
    ScheduleManager::Schedule(4).CurrentValue = 90.0;
```

Reply:
OK, so the schedule values were 0'd out so this is OK.

```
    // Add a test for #7308 without inputs of schedule names
    DataLoopNode::Node(1).OutAirDryBulbSchedNum = 0;
    DataLoopNode::Node(1).OutAirWetBulbSchedNum = 0;
    DataLoopNode::Node(1).OutAirWindSpeedSchedNum = 0;
    DataLoopNode::Node(1).OutAirWindDirSchedNum = 0;
    DataEnvironment::OutDryBulbTemp = 25.0;
    DataEnvironment::OutWetBulbTemp = 20.0;
    DataEnvironment::WindSpeed = 1.5;
    DataEnvironment::WindDir = 90.0;
```

Reply:
Next time think about using unique numbers so there is no ambiguity in the unit test results.

Reply:
OK.